.gitignore

```
@@ -1,5 +1,5 @@
-/db/
 /config/
```

Я бы ещё добавил всю директорию `.idea` в `.gitignore`
```suggestion
/.idea/
/config/
```